integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -245,7 +247,7 @@ public void slaveLost(SchedulerDriver driver, Protos.SlaveID slaveId) {
     public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
       String taskId = status.getTaskId().getValue();
       Protos.TaskState state = status.getState();
-      System.out.printf("Task %s is in state %s\n", taskId, state);
+      System.out.printf("Task %s is in state %s%n", taskId, state);
```

Comment:
`%n` produces the platform-specific newline character, while `\n` is unix-specific.